source/feature/openvswitch/native-openvswitch.html.md

```diff
@@ -25,6 +25,8 @@ OVS is to be fully integrated (natively) into oVirt through VDSM, becoming the d
 ## Overview
 The introduction of OVS as an additional network switching solution raises the need to generalize and abstract the requirements from the system networking services and define a spec which considers both options (OVS & native Linux networking).
 The OVS model of arranging networking entities is different from the native Linux networking, but is closer to industry networking modeling.
+Direct connection of VM to the network cannot be done by OvS. In order to connect a VM to an OvS bridge, a user must first define an OvS VM network. 
+This automatically creates an OVN network, which is connected to OvS one. The OVN network can then be attached to a VM (see [Autodefine External Network](https://ovirt.org/develop/release-management/features/network/autodefine-external-network/)).
```

Comment:
"which is connected to OvS one"
The OVN network is not connected, but rather built on top of OvS.

"The OVN network can then be attached to a VM"
The VM is still connected to the intergration OvS bridge, with an OVN overlay.

Maybe something like (please feel free to improve further):
This automatically creates an OVN network, which is connected to OvS one. The OVN network can then be attached to a VM (see [Autodefine External Network](https://ovirt.org/develop/release-management/features/network/autodefine-external-network/)).
==>
This automatically creates an OVN network on top of the OvS integration bridge. Te VM is is then attached to the OVN network via the OvS integration bridge (see [Autodefine External Network](https://ovirt.org/develop/release-management/features/network/autodefine-external-network/)).

Reply:
Done